include/solvers/mpm_base.tcc

```diff
@@ -187,6 +187,9 @@ bool mpm::MPMBase<Tdim>::initialise_mesh() {
     // Read and assign friction constraints
     this->nodal_frictional_constraints(mesh_props, mesh_io);
 
+    // Create nodal properties
+    mesh_->create_nodal_properties();
```

Comment:
We would have to only do this only if someone has `interface` enabled.

Reply:
Otherwise, we are creating a huge amount of memory unnecessarily. Suggestion is, we create another `MPMExplicit` for now with an option to do interface

Reply:
Hum... Do you mean to crate an entire new MPMExplicit file where the nodal properties are going to be dealt with? Not just create a boolean option to be read from the JSON file, then.

Reply:
Yes, that's correct. We'll discuss a better way later, for now, interface should not affect the main branch.